www/js/controllers/company_home_controller.js

```diff
@@ -0,0 +1,49 @@
+app.controller('CompanyHomeCtrl', function($scope, $state, $ionicLoading, $ionicPopup, BarcodeService, UserService, $ionicHistory) {
+
+  $scope.currentUser = UserService.currentUser();
+
+  if ($scope.currentUser.firstLogin) {
+    $ionicPopup.alert({
+      title: "This device is now associated with " + $scope.currentUser.company + ".",
+      template: "To set your preferences, please see the main menu."
+    });
+  }
+
+  $scope.scanCustomer = function() {
+    $ionicLoading.show();
+
+    BarcodeService.scan()
+    .then(function(id) {
+      UserService.identifyCustomer(id)
+      .then(function() {
+        $scope.customer = UserService.currentCustomer();
+
+        if ($scope.customer.firstPurchase) {
+          $ionicPopup.confirm({
+            templateUrl: "templates/first-purchase.html",
+            scope: $scope,
+            okText: "Confirm"
+          })
+          .then(function(confirmed) {
+            if (confirmed) {
+              $ionicLoading.show();
+              $state.go("app.customer");
+              $ionicLoading.hide();
```

Comment:
Does this need to be here? I thought we were hiding this in the customer home controller once the image has loaded?